net/static_redirect_helper/DEPS

```diff
@@ -0,0 +1,6 @@
+include_rules = [
+  "+brave/browser/safebrowsing",
+  "+brave/components/constants",
+  "+brave/net/geolocation",
+  "+extensions/common",
```

Comment:
so it's not really good to have all these deps added to brave/net level (that is injected into src/net/BUILD.gn), we need to find a better place

Reply:
yes, I agree that `net` does not seem like an appropriate place for this stuff. I'm not sure it's an appropriate place for anything for that matter. Probably components would be better for this. In either case you need to move the code in brave/browser/safebrowsing somewhere else (also probably components) because we should not be including anything from brave/browser in ios.

Reply:
Yeah I wasn't sure where best to put this for cross-platform re-use. I'll look at moving this and the safe browsing build flag to components to address this, thanks!